doc/requirements.txt

```diff
@@ -1,26 +1,5 @@
-alabaster==0.7.7
-argh==0.26.1
-Babel==2.2.0
-backports-abc==0.4
-backports.ssl-match-hostname==3.5.0.1
-certifi==2016.2.28
-CommonMark==0.5.4
-docutils==0.12
-future==0.15.2
-Jinja2==2.11.3
-livereload==2.4.1
-MarkupSafe==0.23
-pathtools==0.1.2
-port-for==0.3.1
-Pygments==2.7.4
-pytz==2016.1
-PyYAML==5.4
-recommonmark==0.4.0
-singledispatch==3.4.0.3
-six==1.10.0
-snowballstemmer==1.2.1
-Sphinx==1.3.3
-sphinx-autobuild==0.6.0
-sphinx-rtd-theme==0.1.9
-tornado==4.3
-watchdog==0.8.3
+recommonmark==0.5.0
+six==1.15.0
+Sphinx==3.3.0
+sphinx-markdown-tables==0.0.9
+sphinx_rtd_theme==0.4.3
```

Comment:
Isn't this `sphinx-rtd-theme`?
cf. https://github.com/start-jsk/jsk_apc/pull/2737/files#diff-246970f36ae7ff82cc458ecc03e34c76cd67f7b50a0e4899b12c31c283b26fcfL24

Reply:
I think either is ok to install in pip.
But OK, I will use `sphinx-rtd-theme`.

Reply:
Oh, sorry, I should have googled...